src/components/pages/InstructorHome/InstructorProfile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Card } from 'antd';
+import { Button } from 'antd';
+
+const { Meta } = Card;
+
+function InstructorProfile() {
+  return (
+    <Card
+      hoverable
+      style={{ width: 240 }}
+      cover={
+        <img
+          alt="example"
```

Comment:
Everything is looking good. We can't be descriptive with alt tags right now, but in the future let's come back and make sure all alt tags have meaningful descriptions.